azafea/event_processors/endless/activation/v1/migrations/81cf792f49b2_remove_region_city.py

```diff
@@ -0,0 +1,48 @@
+# type: ignore
+
+"""Remove region and city and add constraints on latitude and longitude
+
+Revision ID: 81cf792f49b2
+Revises: 3ce72dccdef8
+Create Date: 2021-06-17 12:49:31.415945
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '81cf792f49b2'
+down_revision = '3ce72dccdef8'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_column('activation_v1', 'city')
+    op.drop_column('activation_v1', 'region')
+
+    op.execute(
+        'UPDATE activation_v1 \n'
+        'SET latitude = CASE WHEN latitude=90 THEN 89.5 ELSE floor(latitude)+0.5 END;'
+    )
+    op.execute(
+        'UPDATE activation_v1 \n'
+        'SET longitude = CASE WHEN longitude=180 THEN 179.5 ELSE floor(longitude)+0.5 END;'
+    )
```

Comment:
I am not a postgresql expert but I believe that when one updates a row, in fact postgresql writes a new row and marks the old row as garbage. As written, this migration will rewrite the whole table to update latitude, and then immediately rewrite the whole table again to update longitude, generating another 1 650 991 rows of garbage. Is it worth combining the two `UPDATE` statements into one?